dotnet-desktop-guide/framework/winforms/controls/how-to-show-a-color-palette-with-the-colordialog-component.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Show a Color Palette with the ColorDialog Component"
+description: Learn how to show a color palette with the ColorDialog component, which displays a palette of colors and returns a property containing the color the user has selected.
```

Comment:
This one's over the 160 character limit. I'd shorten it slightly to: "Learn how to show a color palette with the ColorDialog component, which displays a palette of colors and returns a property containing the user-selected color."

Reply:
Addressed.